package.json

```diff
@@ -90,5 +90,8 @@
     "q": "1.4.1",
     "u2f-api-polyfill": "0.4.1",
     "underscore": "1.8.3"
+  },
+  "optionalDependencies": {
+    "fsevents": "*"
```

Comment:
why do we need the fsevents here? 
We've had problems with this before, so at some point we removed it from npm-shrinkwrap. Then I think someone checked it back in and builds have been working fine so far...